webapp/security/schemas.py

```diff
@@ -84,7 +84,7 @@ class NoticePackage(Schema):
 
 
 class NoticeSchema(Schema):
-    id = String(required=True, validate=Regexp(r"USN-\d{1,5}-\d{1,2}"))
+    id = String(required=True, validate=Regexp(r"(LSN-|USN-)\d{1,5}-\d{1,2}"))
```

Comment:
why not

```suggestion
    id = String(required=True, validate=Regexp(r"(LSN|USN)-\d{1,5}-\d{1,2}"))
```

?

I'm gonna approve this anyway, this is just a suggestion.

Reply:
@nottrobin no good reason.. :D  I used as an example the regex we used for cves where the - is included. :P

Reply:
Done.